### DISCUSSION TITLE
Is it possible to contribute to the docs?

### DISCUSSION THREAD

**Discussion by bojanth** [artilleryio/artillery#2291] [Artillery]
Hello!
There is a small typo in the [docs](https://www.artillery.io/docs/reference/test-script#load-phase---additional-options), `mavVusers` should be change to `maxVusers`.
I've searched the repositories to open a PR but couldn't find the repository for the docs.

**Comment by bernardobridge** [comment]
Hi @bojanth . The docs are in a private repo, so unfortunately that's not currently possible. We may change that in the future!

Thanks for reporting this though, I'll fix go ahead and fix it! 🙇

**Reply by bernardobridge** [reply]
Done: https://www.artillery.io/docs/reference/test-script#load-phase---additional-options

**Reply by bojanth** [reply]
Thank you for the quick response and the fix!
